### ISSUE TITLE
[Content] Adding a webscraper

### ISSUE THREAD

**Issue by Jeffrey2659** [gracekishino/coding_playlists#24]
Could I add a webscraper, to automatically update the site with coding videos from youtube or other websites?

**Comment by Jeffrey2659** [comment]
It would be done using javascript

**Comment by gracekishino** [comment]
@Jeffrey2659 yes this sounds like a great idea! I think we need to be careful of copyright, but assume that it's OK with sites like YouTube and Spotify where they allow sharing playlists and embedding on other sites.
